### ISSUE TITLE
Implement Jitsi API Command Execution Function

### ISSUE THREAD

**Issue by antonplagemann** [Music-Bot-for-Jitsi/jitsi-api-client#4]
> **As a** developer
> **I want** to have a function that executes commands at the jitsi api
> **so that** I do user actions programmatically.

## Description:
Implement functions so that different commands can be executed on the jitsi api.

### 🟢 In scope:
- [ ] Function and methods needed to execute commands at the jitsi api

### 🔴 Not in scope:
- Event pass-through

## What should be the result?
A functions that can execute one or more commands at the jitsi api.

**Comment by antonplagemann** [comment]
Blocked by #3

**Comment by antonplagemann** [comment]
Blocked: Continuing on the alternative solution with headless Chrome for now.
